### ISSUE TITLE
real-world dataset training and evaluation

### ISSUE THREAD

**Issue by lx709** [junzhezhang/shape-inversion#9]
Dear Dr. Zhang, Thanks for sharing your interesting work. I'm currently working on the same task and want to compare the performance with your ShapeInversion model. I'd like to know when training and evaluating your model on a real-world dataset, are you sampling the same number of 2048 points for each shape?

**Comment by junzhezhang** [comment]
Hi, 
Thanks for your interest in our work. Yes, we follow the setting in pcl2pcl and sample the same number of 2048 points in each partial shape. However, our framework is robust to the number of points in the partial shape. If you happen to work on partial shapes with fewer points, you may tune down the value k in the K-mask. 

Beside, when we conduct experiments on real-world dataset, our pre-trained GAN is still trained on CRN dataset, which is derived from ShapeNet.

Thanks,
Junzhe

**Comment by lx709** [comment]
Dear Dr. Zhang, 
    Thanks for your quick reply. In the response, you wrote, "when we conduct experiments on real-world dataset, our pre-trained GAN is still trained on CRN dataset" (strategy 1). This actually confuses me. Because in the supplementary of your paper, it said: "Both ScanNet [5] and KITTI [6] are split into train set and test set, and the mapping GAN of the pcl2pcl framework, which maps the latent space of partial shapes to that of the complete ones, is retrained on the real scan train set" (strategy 2). I've also tried to directly transfer the model trained on CRN to real-world datasets but got degraded performance. For a fair comparison, I'd like to know which strategy you tried in the original paper. Thanks for your attention.
Best regards,
Xiang

**Comment by junzhezhang** [comment]
Hi Xiang,
The description in the supplementary actually refers on which data split both ShapeInversion and pcl2pcl are evaluated, and how pcl2pcl is trained. Note that ShapeInversion uses the same pre-trained model across all experiments, which are trained on CRN train split. This is described in Implementation Details of the main paper. This shows the remarkable generalization of ShapeInversion.
Hope it clarifies.

Best,
Junzhe

**Comment by lx709** [comment]
Dear Dr. Zhang,
    Thanks for your kind clarification. It helps a lot!
Best,
Xiang
